reana_workflow_controller/rest.py

```diff
@@ -24,23 +24,21 @@
                                               WORKFLOW_TIME_FORMAT)
 from reana_workflow_controller.errors import (REANAWorkflowControllerError,
                                               UploadPathError,
+                                              WorkflowDeletionError,
                                               WorkflowInexistentError,
                                               WorkflowNameError)
 from reana_workflow_controller.tasks import (run_cwl_workflow,
                                              run_serial_workflow,
                                              run_yadage_workflow)
 from reana_workflow_controller.utils import (create_workflow_workspace,
-                                             list_directory_files)
+                                             list_directory_files,
+                                             remove_workflow_workspace)
 
 START = 'start'
 STOP = 'stop'
 PAUSE = 'pause'
-STATUSES = {START, STOP, PAUSE}
-
-workflow_spec_to_task = {
-    "yadage": run_yadage_workflow,
-    "cwl": run_cwl_workflow
-}
+DELETED = 'deleted'
```

Comment:
Shall we improve this and use the [WorkflowStatus enum](https://github.com/reanahub/reana-db/blob/master/reana_db/models.py#L56-L62) instead of hardcoding this strings here? We could do `[s.name for s in WorkflowStatus]`.

Reply:
Yes I agree, since this was not introduced in this PR I'll create a new issue.